saleor/graphql/product/types.py

```diff
@@ -35,17 +46,32 @@ def resolve_attribute_list(attributes):
     return attributes_list
 
 
+def resolve_attribute_value_type(attribute_value):
+    color_pattern = re.compile(COLOR_PATTERN)
```

Comment:
You still need to compile the regex at the top of the file, then you will use the compiled regex as is. Otherwise it doesn't make sense to use `re.compile`.